flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -287,6 +287,13 @@
             CONSOLE_USER_EMAIL
     );
 
+    public static final UnboundBooleanFlag CONFIGSERVER_PROVISION_LB = defineFeatureFlag(
+            "configserver-provision-lb", false,
+            "Provision load balancer for config server cluster",
+            "Takes effect when zone-config-servers application is redeployed",
+            ZONE_ID
```

Comment:
This is likely not necessary as this code will run on the config server? The zone_id dimension is only relevant on flags used in the controller.

Reply:
Yes, but I want to use the same flag in the controller when implementing DNS updates.